src/styles/blocks/button.scss

```
@@ -2,16 +2,17 @@
   display: flex;
   justify-content: center;
   background: $page-color;
+  border: 1px solid $page-color;
+  border-radius: 5px;
   color: #fff;
   @include font(14, 700);
-  line-height: 40px;
+  line-height: 38px;
   text-transform: uppercase;
   text-decoration: none;
-  border-radius: 5px;
+  transition: 0.4s;
```

Always list styles that you apply transition for;
```suggestion
  transition: color 0.4s, background-color 0.4s;
```